### PR TITLE
Topics to dashboad (routes + menu label)

### PR DESCRIPTION
**Pull request by dancer2090**
## Overview
This PR adds "dashboards" label to main menu instead of "topic" and updates routes from /topics to /dashboards
Topics page still available by route /topics.
All changes with dashboard content have done in topics components. (layout/topics and layout/app/topics-detail). We are planning to rename them in future PR.

## Testing instructions
From main page. Click "dashboards" label in main menu or go to /dashboards route. You should see the content from previous "topics" page.
Click on some topic on this page or follow the link http://localhost:9000/dashboards/climate

## [Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/169732634)
---

## Checklist before submitting
- [x] Title: Don't forget to make it clear for everyone, even for people that don't know about the feature/bug.
- [x] Meaningful commits and code rebased on `develop`.

### CODE REVIEW

**Comment by dancer2090** [comment]
> 1. I'm still seeing the word `Topics` in the page title point_down
> 
> image
> 
> 1. In the former `Topic detail` page the section `Other topics` should read `Other dashboards` instead point_down
> 
> image
> 
> 1. We shouldn't duplicate the existing classes and files that already exist for topics in this PR, better to refactor those classes and components instead. We can create later a different PR consisting of renaming all those files so that they are named _dashboard(s)_.

Fixed. 
1) The title on /dashboards page has changed to 'Dashboard | Resource Watch'
2) Other topics on /dashboard/id page has changed to 'Other dashboards'. The links below this title follows /dashboards/id page
3) All changes with dashborads content have done in topics components (without renaming them). We are planning to rename them in future PR.
4) Footer menu has been updated. Removed topics items like on layout: https://projects.invisionapp.com/d/main#/console/11337456/243102360/preview

File: layout/footer/constants.js [hunk 1]
```diff
@@ -15,28 +15,6 @@ export const FOOTER_LINKS = [
       }
     ]
   },
-  {
```
Comment (pablopareja) [inline layout/footer/constants.js]: @dancer2090 Can we keep this section and rename it to `Dashboards`?

File: layout/header/constants.js [hunk 2]
```diff
@@ -16,19 +16,19 @@ export const APP_HEADER_ITEMS = [
...
   },
   {
-    id: 'topics',
-    label: 'Topics',
-    route: 'topics',
-    pathnames: ['/app/topics', '/app/topics-detail'],
+    id: 'dashboards',
+    label: 'Dashboards',
+    route: 'dashboards',
+    pathnames: ['/app/About', '/app/Partners'],
```
Comment (pablopareja) [inline layout/header/constants.js]: @dancer2090 I don't understand this change, why did you set `About` and `Partners` as pathnames?

File: layout/topics/component.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ class TopicsLayout extends PureComponent {
 
     return (
       <Layout
-        title="Topics — Resource Watch"
+        title="Dashboards"
```
Comment (pablopareja) [inline layout/topics/component.js]: @dancer2090 Let's keep the `- Resource Watch` sufix after `Dashboards`

File: pages/app/dashboards-detail/index.js [hunk 4]
```diff
@@ -1,21 +1,58 @@
 import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
 
 // actions
-import { getDashboard } from 'modules/dashboards/actions';
```
Comment (pablopareja) [inline pages/app/dashboards-detail/index.js]: @dancer2090 Your changes in this page have broken the dashboard preview pages, here's a url example where you can check that http://localhost:9000/data/dashboards/test-dashboard-is_highlighted

Comment (pablopareja) [reply pages/app/dashboards-detail/index.js]: If you want to check it with one of your dashboards, go to `MyRW` --> `Dashboards` and you will see the preview links in the cards.
image

**Comment by dancer2090** [comment]
All changes were moved to 
https://github.com/resource-watch/resource-watch/tree/feature/topics-to-dashboard 
branch

**Pull request closed without merging**